<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MasterRouteWidget</class>
 <widget class="QWidget" name="MasterRouteWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="mrsLoadPage">
      <layout class="QGridLayout" name="gridLayout_11">
       <item row="0" column="0">
        <widget class="QPushButton" name="loadRoutesButton">
         <property name="text">
          <string>Load Routes</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mrsWaitPage">
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="0">
        <widget class="QLabel" name="bufferPleaseWaitLabel">
         <property name="font">
          <font>
           <pointsize>22</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Please Wait</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="abortButton">
         <property name="text">
          <string>Abort!</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mrsRoutePage">
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Monday</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="1">
            <widget class="QStackedWidget" name="mondayMRSStack"/>
           </item>
           <item row="1" column="0">
            <widget class="QListView" name="mondayMRSView"/>
           </item>
           <item row="0" column="0">
            <widget class="QLineEdit" name="mondayRouteSearch">
             <property name="text">
              <string>Search</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Tuesday</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="0">
            <widget class="QListView" name="tuesdayMRSView"/>
           </item>
           <item row="1" column="1">
            <widget class="QStackedWidget" name="tuesdayMRSStack"/>
           </item>
           <item row="0" column="0">
            <widget class="QLineEdit" name="tuesdayRouteSearch">
             <property name="text">
              <string>Search</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>Wednesday</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="1" column="1">
            <widget class="QStackedWidget" name="wednesdayMRSStack"/>
           </item>
           <item row="1" column="0">
            <widget class="QListView" name="wednesdayMRSView"/>
           </item>
           <item row="0" column="0">
            <widget class="QLineEdit" name="wednesdayRouteSearch">
             <property name="text">
              <string>Search</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>Thursday</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="1" column="1">
            <widget class="QStackedWidget" name="thursdayMRSStack"/>
           </item>
           <item row="1" column="0">
            <widget class="QListView" name="thursdayMRSView"/>
           </item>
           <item row="0" column="0">
            <widget class="QLineEdit" name="thursdayRouteSearch">
             <property name="text">
              <string>Search</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_5">
          <attribute name="title">
           <string>Friday</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="1" column="0">
            <widget class="QListView" name="fridayMRSView"/>
           </item>
           <item row="1" column="1">
            <widget class="QStackedWidget" name="fridayMRSStack"/>
           </item>
           <item row="0" column="0">
            <widget class="QLineEdit" name="fridayRouteSearch">
             <property name="text">
              <string>Search</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_6">
          <attribute name="title">
           <string>Saturday</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="1" column="1">
            <widget class="QStackedWidget" name="saturdayMRSStack"/>
           </item>
           <item row="1" column="0">
            <widget class="QListView" name="saturdayMRSView"/>
           </item>
           <item row="0" column="0">
            <widget class="QLineEdit" name="saturdayRouteSearch">
             <property name="text">
              <string>Search</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_7">
          <attribute name="title">
           <string>Sunday</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="1" column="0">
            <widget class="QListView" name="sundayMRSView"/>
           </item>
           <item row="1" column="1">
            <widget class="QStackedWidget" name="sundayMRSStack"/>
           </item>
           <item row="0" column="0">
            <widget class="QLineEdit" name="sundayRouteSearch">
             <property name="text">
              <string>Search</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="refreshMRSButton">
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
